Sources/Overload/OvCore/src/OvCore/ECS/Components/CPostProcessStack.cpp

```diff
@@ -0,0 +1,176 @@
+
+	OvCore::Helpers::GUIDrawer::DrawBoolean(p_root, "Bloom Enabled", bloomSettings.enabled);
+	OvCore::Helpers::GUIDrawer::DrawScalar<float>(p_root, "Bloom Threshold", bloomSettings.threshold, 0.05f, 0.0f);
+	OvCore::Helpers::GUIDrawer::DrawScalar<float>(p_root, "Bloom Radius", bloomSettings.radius, 0.1f, 0.0f);
+	OvCore::Helpers::GUIDrawer::DrawScalar<int>(p_root, "Bloom Kernel Size", bloomSettings.kernelSize, 1.0f, 1, 10);
+	OvCore::Helpers::GUIDrawer::DrawScalar<float>(p_root, "Bloom Intensity", bloomSettings.intensity, 0.05f, 0.0f);
+	OvCore::Helpers::GUIDrawer::DrawScalar<int>(p_root, "Bloom Passes", bloomSettings.passes, 1, 1, 10);
+	
+	p_root.CreateWidget<OvUI::Widgets::Visual::Separator>();
+	p_root.CreateWidget<OvUI::Widgets::Layout::Spacing>();
+
+	OvCore::Helpers::GUIDrawer::DrawBoolean(p_root, "Auto Exposure Enabled", autoExposureSettings.enabled);
+	OvCore::Helpers::GUIDrawer::DrawScalar<float>(p_root, "Auto Exposure Center Weight Bias", autoExposureSettings.centerWeightBias, 0.1f, 0.0f, 1.0f);
+	OvCore::Helpers::GUIDrawer::DrawScalar<float>(p_root, "Auto Exposure Min Luminance (EV)", autoExposureSettings.minLuminanceEV, 1.0f, OvCore::Helpers::GUIDrawer::_MIN_FLOAT, OvCore::Helpers::GUIDrawer::_MAX_FLOAT);
+	OvCore::Helpers::GUIDrawer::DrawScalar<float>(p_root, "Auto Exposure Max Luminance (EV)", autoExposureSettings.maxLuminanceEV, 1.0f, OvCore::Helpers::GUIDrawer::_MIN_FLOAT, OvCore::Helpers::GUIDrawer::_MAX_FLOAT);
+	OvCore::Helpers::GUIDrawer::DrawScalar<float>(p_root, "Auto Exposure Exposure Compensation (EV)", autoExposureSettings.exposureCompensationEV, 1.0f, OvCore::Helpers::GUIDrawer::_MIN_FLOAT, OvCore::Helpers::GUIDrawer::_MAX_FLOAT);
+	OvCore::Helpers::GUIDrawer::DrawBoolean(p_root, "Auto Exposure Progressive", autoExposureSettings.progressive);
+	OvCore::Helpers::GUIDrawer::DrawScalar<float>(p_root, "Auto Exposure Speed Up", autoExposureSettings.speedUp);
+	OvCore::Helpers::GUIDrawer::DrawScalar<float>(p_root, "Auto Exposure Speed Down", autoExposureSettings.speedDown);
+
+	p_root.CreateWidget<OvUI::Widgets::Visual::Separator>();
+	p_root.CreateWidget<OvUI::Widgets::Layout::Spacing>();
+
+	OvCore::Helpers::GUIDrawer::DrawBoolean(p_root, "Tonemapping Enabled", tonemappingSettings.enabled);
+	OvCore::Helpers::GUIDrawer::DrawScalar<float>(p_root, "Tonemapping Exposure", tonemappingSettings.exposure, 0.05f, 0.0f);
+	Helpers::GUIDrawer::CreateTitle(p_root, "Tonemapping Mode");
+	auto& tonemappingMode = p_root.CreateWidget<OvUI::Widgets::Selection::ComboBox>(static_cast<int>(tonemappingSettings.mode));
+	tonemappingMode.choices.emplace(0, "Neutral");
+	tonemappingMode.choices.emplace(1, "Reinhard");
+	tonemappingMode.choices.emplace(2, "Reinhard-Jodie");
+	tonemappingMode.choices.emplace(3, "Uncharted 2");
+	tonemappingMode.choices.emplace(4, "Uncharted 2 (Filmic)");
+	tonemappingMode.choices.emplace(5, "ACES");
```

Comment:
Could be replaced by :
```suggestion
	tonemappingMode.choices = {
		{0, "Neutral"},
		{ 1, "Reinhard" },
		{ 2, "Reinhard-Jodie" },
		{ 3, "Uncharted 2" },
		{ 4, "Uncharted 2 (Filmic)" },
		{ 5, "ACES" }
	};
```

Reply:
Fair enough!

Reply:
Can you add a space before the `0` 😅